src/Gui/SettingsScreen/LunchBreak.java

```diff
@@ -0,0 +1,76 @@
+package Gui.SettingsScreen;
+
+import Gui.Dropdown;
+import javafx.beans.value.ChangeListener;
+import javafx.beans.value.ObservableValue;
+import javafx.event.Event;
+import javafx.event.EventHandler;
+import javafx.scene.control.Label;
+import javafx.scene.control.Spinner;
+import javafx.scene.control.SpinnerValueFactory;
+import javafx.scene.layout.HBox;
+import javafx.util.Pair;
+
+public class LunchBreak {
+    //BreakLength means the amount of minutes a break takes
+    //BreakTime means after which lesson a break starts
+    private Label label;
+    private Spinner<Integer> BLSpinner;
+    private Dropdown BTDropdown;
```

Comment:
Properties zouden niet met een hoofdletter moeten beginnen.

Gebruik hier
```suggestion
    private Dropdown breakTimeDropdown;
```
of
```suggestion
    private Dropdown btDropdown;
```

Hetzelfde geld voor `BLSpinner`, `BLCurrent`, `BLMemory`, `BTCurrent` en `BTMemory`

Reply:
on it!